spring-ai-alibaba-graph/spring-ai-alibaba-graph-studio/src/main/java/com/alibaba/cloud/ai/service/dsl/nodes/VariableAggregatorNodeDataConverter.java

```diff
@@ -0,0 +1,76 @@
+package com.alibaba.cloud.ai.service.dsl.nodes;
+
+import com.alibaba.cloud.ai.model.workflow.NodeType;
+import com.alibaba.cloud.ai.model.workflow.nodedata.VariableAggregatorNodeData;
+import com.alibaba.cloud.ai.service.dsl.NodeDataConverter;
+import com.alibaba.fastjson.JSON;
+import lombok.SneakyThrows;
+import org.springframework.stereotype.Component;
+import org.springframework.util.ReflectionUtils;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Component
+public class VariableAggregatorNodeDataConverter implements NodeDataConverter<VariableAggregatorNodeData> {
+
+    @Override
+    public Boolean supportType(String nodeType) {
+        return NodeType.AGGREGATOR.value().equals(nodeType);
+    }
+
+
+    @Override
+    public VariableAggregatorNodeData parseDifyData(Map<String, Object> data) {
+        VariableAggregatorNodeData variableAggregatorNodeData = new VariableAggregatorNodeData(null, null);
+        variableAggregatorNodeData.setTitle((String) data.get("title"));
+        variableAggregatorNodeData.setType((String) data.get("type"));
+        variableAggregatorNodeData.setSelected((Boolean) data.get("selected"));
+        variableAggregatorNodeData.setOutput_type((String) data.get("output_type"));
+        variableAggregatorNodeData.setDesc((String) data.get("desc"));
+        variableAggregatorNodeData.setVariables((List<List<String>>) data.get("variables"));
+        Map<String,Object> advanced_settings = (Map<String, Object>) data.get("advanced_settings");
+        VariableAggregatorNodeData.AdvancedSettings advancedSettings  = new VariableAggregatorNodeData.AdvancedSettings();
```

Comment:
Maybe multiple `setXXX`  calls  can be simplified as `setTitle().setType().setXXX()`?